plugins/jobs/web_client/JobStatus.js

```diff
@@ -79,19 +84,22 @@ JobStatus.registerStatus({
         value: 3,
         text: 'Success',
         icon: 'icon-ok',
-        color: '#53b653'
+        color: '#53b653',
+        finished: true
```

Comment:
Rather than rely on the other statuses to have `undefined` -> `false` values of `finished`, I think we should explicitly define `finished: false` on them.

We can then more cleanly decide if we want custom / unknown status types to have a default `finished` value of `true` or `false`.